documentation/3_Layers_of_the_Reference_Architecture_Model/3_3_Process_Layer/3_3_5_Publishing_and_using_Data_Apps.md

```diff
@@ -1,8 +1,23 @@
-## PUBLISHING AND USING DATA APPS
-Data Apps can be used by Connectors for specific data processing or data transformation tasks. They can perform tasks of different complexity, ranging from simple data transformation to complex data analytics. An example of data transformation may be a Data App parsing a single string field with address information and producing a data structure consisting of street name and number, zip code, name of the city, and name of the country.
+## Publishing and using Data Apps
```

Comment:
@kfidomski As discussed, we should call it IDS Apps instead of Data Apps if we mean all types of IDS Apps. Can you adjust the files, content and figures accordingly?

Reply:
Addressed in commit https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/pull/145/commits/9b5df2cd07d1221f1c5ea2c8e8f53a9cf53590e6 and https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/pull/145/commits/00996cd974b30dd365434dd7bf039a8a9e1257e8